pilot/pkg/networking/core/v1alpha3/listener_inbound.go

```diff
@@ -271,11 +271,16 @@ func (lb *ListenerBuilder) buildInboundChainConfigs() []inboundChainConfig {
 				Protocol:   i.ServicePort.Protocol,
 			}
 
+			bindAddr := WildcardAddress
+			if lb.node.IsIPv6() {
+				bindAddr = WildcardIPv6Address
+			}
+
 			cc := inboundChainConfig{
 				telemetryMetadata: telemetry.FilterChainMetadata{InstanceHostname: i.Service.Hostname},
 				port:              port,
 				clusterName:       model.BuildInboundSubsetKey(int(port.TargetPort)),
-				bind:              "0.0.0.0", // TODO ipv6
```

Comment:
So this is being changed for every service not just service entry based clusters?

Reply:
Yes, @ramaraochavali, it's being changed for every service. However, `lb.node.IsIPv6()` is different with `node.SupportIPv6`, the former checks that the node is IPv6 only, and the latter checks that the node can be both IPv6 only or Dual. 
So this change only impacts the IPv6 only cluster.

Reply:
got it